src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -474,4 +499,16 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 		this.cellModel.active = true;
 		this._model.updateActiveCell(this.cellModel);
 	}
+
+	private findPathRelativeToContent(elementContent: string): string {
```

Comment:
Seems like this shouldn't be hard to add a few tests for

Reply:
I was thinking about this when I woke up today. Let me get this in so that it makes the release branch and write some tests later today. Stand by 😄